src/app/tests/suites/certification/Test_TC_APPLAUNCHER_3_7.yaml

```diff
@@ -57,16 +60,34 @@ tests:
           values:
               - name: "Status"
                 value: 0
+              - name: "Data"
+                value: "DataValue"
 
     - label:
-          "Step 2: TH sends a LaunchApp command to DUT to launch an app which is
-          not available"
-      PICS: APPLAUNCHER.S.C00.Rsp
+          "Step 1b: TH reads the Status attribute from application Basic cluster"
+      verification: |
+          ./chip-tool applicationbasic read status 1 3
+           On the TH (chip-tool), verify that the status is 1, which indicates the current running state of the application (1 = running, visible to the user, and the active target for input)           [1646143551275] [87039:3765756] CHIP: [TOO] Endpoint: 3 Cluster: 0x0000_050D Attribute 0x0000_0005DataVersion: 4144987298
```

Comment:
If no app has been launched yet, then there may not be an endpoint 3.

Reply:
got it. It would be great to make 3 a variable defined at the top of the script (content_app_endpoint_no) or something like that. It will often by 3 but not guaranteed.